sensor/common/resources/metadatagetter/datastore_impl_test.go

```diff
@@ -0,0 +1,181 @@
+package metadatagetter
```

Comment:
memstore instead of datastore in filename for this file and corresponding test file